doc/teacher_doc/course_admin.rst

```diff
@@ -70,30 +70,26 @@ Task edition
 ------------
 
 All tasks can be edited from the webapp. To access the task editor, just click
-on *Edit task* on the task page or from the main administration page.
+on *Tasks* from the main administration page. Then click on *Edit task* for the concerned task. 
+You can also add new tasks from this *Tasks* page by clicking *Add tasks* for a tasks section and entering a new task id.
 
-Task problems containing boxes are not graphically editable due to
-their high modularity. These kinds of problem editable on-line in YAML
-format.
 
-Adding/removing problems
+Adding/removing subproblems
 ````````````````````````
-Adding and removing problems are very easy in the task editor, go to the
-end of the page or click on the quick link "Add subproblem". You'll then
-be brought to a new form asking a problem-id (alphanumerical characters)
-and a problem type.
-
-To make a more complex question with boxes, choose "custom" problem and
-write the YAML problem description as described in the task file format.
+Adding and removing subproblems is very easy in the task editor. Go to the *Subproblems* tab and add a new 
+problem-id (alphanumerical characters) and a problem type. You can configure the problem context from this page, 
+even finalize the entire problem for mcq or match problems.
 
 When editing a multiple choice problem, you're asked if the student is
 shown a multiple-answers- or single-answer-problem and which of the
 possible choices is (are) good answer(s).
 
+Only more complex problems such as code implementation will require more configuration with a grading script in the task files.
```

Comment:
It would be better to specify that there currently are two types of grading environment:
- MCQ+ based, which call the internal grading function
- Container-based, which allows to define your own grading script and gives access to the user inputs.

And that among the default subproblems that are shipped with a typical INGInious installation, those two types have an internal grading function:
- MCQ
- Match problems

Maybe this is better to rename or split the section to make this more clear.

Reply:
Probably better now.

Reply:
Well, it's a bit better, but actually you would have more chance to copy/paste my comment :D The documentation should explicitely tell that those two features are possible for each kind of subproblem, but that in the bundled subproblems type, only mcq and match problems support both. This is important because some other available subproblems (math, network-trace,...) support both.